model/src/main/java/org/pentaho/database/model/DatabaseConnection.java

```diff
@@ -668,18 +720,18 @@ public String getId() {
 
   @Override
   public String toString() {
-    return "DatabaseConnection [id=" + id + ", name=" + name + ", databaseName=" + databaseName + ", databasePort="
-        + databasePort + ", hostname=" + hostname + ", username=" + username + ", password=*****"
-        + ", dataTablespace=" + dataTablespace + ", indexTablespace=" + indexTablespace + ", streamingResults="
-        + streamingResults + ", quoteAllFields=" + quoteAllFields + ", changed=" + changed
-        + ", usingDoubleDecimalAsSchemaTableSeparator=" + usingDoubleDecimalAsSchemaTableSeparator
-        + ", informixServername=" + informixServername + ", forcingIdentifiersToLowerCase="
-        + forcingIdentifiersToLowerCase + ", forcingIdentifiersToUpperCase=" + forcingIdentifiersToUpperCase
-        + ", connectSql=" + connectSql + ", usingConnectionPool=" + usingConnectionPool + ", accessTypeValue="
-        + accessTypeValue + ", accessType=" + accessType + ", databaseType=" + databaseType + ", extraOptions="
-        + extraOptions + ", attributes=" + attributes + ", connectionPoolingProperties=" + connectionPoolingProperties
-        + ", partitioningInformation=" + partitioningInformation + ", initialPoolSize=" + initialPoolSize
-        + ", maxPoolSize=" + maxPoolSize + ", partitioned=" + partitioned + "]";
+    return "DatabaseConnection [name=" + name + ", databaseName=" + databaseName + ", databasePort="
```

Comment:
Why did you remove ID field from toString()?

Reply:
Yes, by mistake